src/models/mods.rs

```diff
@@ -1,5 +1,71 @@
+use chrono::{DateTime, Utc};
 use serde::{Deserialize, Serialize};
 
+// TODO: use serde to parse this as base62
+#[derive(Serialize, Deserialize)]
+#[serde(transparent)]
+pub struct ModId(pub String);
+
+// TODO: use serde to parse this as base62
+#[derive(Serialize, Deserialize)]
+#[serde(transparent)]
+pub struct UserId(String);
+
+// TODO: what format should this be?
+#[derive(Serialize, Deserialize)]
+#[serde(transparent)]
+pub struct VersionId(String);
+
+// TODO: permissions, role names, etc
+#[derive(Serialize, Deserialize)]
+pub struct Team {
+    users: Vec<(String, UserId)>,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct Mod {
+    pub id: ModId,
+    pub team: Team,
+
+    pub title: String,
+    pub description: String,
+    pub published: DateTime<Utc>,
+
+    pub downloads: u32,
+    pub categories: Vec<String>,
+    pub versions: Vec<VersionId>,
+
+    pub body_url: String,
```

Comment:
what does this do

Reply:
I don't really know, it's from the database model.  I would assume that it's a link to the main page of the mod, which doesn't make sense anymore with the API being separated from the front end.

Reply:
I think its safe to not include this on the schema then